docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -82,7 +82,42 @@ Disable bootstrapping enrollment by setting `ZITI_BOOTSTRAP_ENROLLMENT=false` in
 
 ## Firewall
 
-The router's generated configuration always listens on a single, configurable TCP port (default: `3022`). You may set `ZITI_ROUTER_PORT` in `/opt/openziti/etc/router/bootstrap.env` to bootstrap with a different port.
+The router's generated configuration always uses a single TCP port (default: `3022`) defined by `ZITI_ROUTER_PORT` in `/opt/openziti/etc/router/bootstrap.env`.
+
+Confirm the router is running and listening on the expected port.
+
+```bash
+systemctl show -p MainPID --value ziti-router.service \
+| xargs -rIPID sudo lsof -Pnp PID \
+|& grep LISTEN
+```
+
+```buttonless title="Output"
+ziti    3574455 ziti-router    7u     IPv6           41543151       0t0      TCP *:3022 (LISTEN)
+```
+
+Substitute the router's advertised address and port for `localhost:3022` as defined in `/var/lib/ziti-router/config.yml`.
```

Comment:
"3022" should be replaced with whatever their port is  and should be referenced here to remind them that it's not always 3022